mlflow/transformers/__init__.py

```
@@ -2761,6 +2802,32 @@ def is_uri(s):
                 error_code=BAD_REQUEST,
             )
 
+    def _wrap_input_in_prompt_template(self, input_data):
+        """
+        Wraps the input data in the specified prompt template. If no template is
+        specified, or if the pipeline is an unsupported type, or if the input type
+        is not a string or list of strings, then the input data is returned unchanged.
+        """
+
+        # if no template is set, do nothing
+        if not self.prompt_template:
+            return input_data
+
+        if self.pipeline.task not in _SUPPORTED_PROMPT_TEMPLATING_TASK_TYPES:
```

definitely raise here instead of silently passing.